packages/git/src/node/dugite-git.spec.ts

```diff
@@ -73,23 +74,26 @@ describe('git', async function (): Promise<void> {
 
         });
 
-        it('should discover all nested repositories and the root repository which is at the workspace root', async () => {
+        // https://github.com/eclipse-theia/theia/issues/933
+        if (!isWindows) {
```

Comment:
Just in case you are not aware of it:
https://github.com/eclipse-theia/theia/blob/8cf1deca3db2ae587942725b3fb68f67a8eab32b/packages/filesystem/src/node/node-filesystem.spec.ts#L370-L374

If you `skip` the tests like this, the number of tests won't differ between Windows and Linux/macOS, but it will be reported as skipped.

Reply:
I initially tried this way but it resulted in a failure ".skip() does not exist on type Suite":
https://github.com/eclipse-theia/theia/commit/077394e3eb16a9df9b9a64e3e62ba9afc48d84f3

Reply:
Because you need explicitly write `async function () {` instead of `async () => {`.
It is the same as when you would like to set a `timeout`.
For more details, see https://github.com/mochajs/mocha/issues/2018#issue-123301524.

Let me know if it still does not work.

Reply:
And here is the doc from Mocha: https://mochajs.org/#arrow-functions

Reply:
Thank you for your help @kittaakos, I've updated the code following your suggestions.